app/shop.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+from typing import Any
+
+
+class Shop:
+    def __init__(self, shop_name: str, location: list, products: dict) -> None:
+        self.shop_name = shop_name
+        self.location = location
+        self.products = products
+
+    @staticmethod
+    def format_price(price: float) -> str:
+        return f"{price: .2f}".rstrip("0").rstrip(".")
+
+    def print_receipt(self, customer: Any) -> None:

```

Comment:
```suggestion
    def print_receipt(self, customer: Customer) -> None:

```

Reply:
If I import and use here customer: Customer I will get
ImportError: cannot import name 'Shop' from partially initialized module 'app.shop' (most likely due to a circular import) (C:\Users\blink\mate_academy_projects\py-shop-trip\app\shop.py)
![Desktop Screenshot 2024 07 24 - 10 35 15 05](https://github.com/user-attachments/assets/3f86b514-2a0c-416e-b158-ea53ece74c32)

Honestly, I don't know how to fix it, maybe some tips on how to solve this problem